neuron_morphology/transforms/affine_transform.py

```diff
@@ -160,14 +160,19 @@ def _get_scaling_factor(self) -> float:
         determinant = np.linalg.det(self.affine)
         return np.power(abs(determinant), 1.0 / 3.0)
 
-    def transform_morphology(self, morphology: Morphology,
-                             clone: bool = False) -> Morphology:
+    def transform_morphology(self,
+                             morphology: Morphology,
+                             clone: bool = False,
+                             scale_radius: bool = True,
```

Comment:
Given what nathan said, we may want to default this to False

Reply:
Good point, but he only mentioned the scale correction, and I would prefer sorting this out with him to be sure. It should not affect over transforms, but we should get back to this fine point.